docs/java/signals.md

```diff
@@ -3,3 +3,61 @@ id: signals
 title: Signals in Java
 sidebar_label: Signals
 ---
+
+Workflows can listen to external events and save the state accordingly.
+
+```java
+import io.temporal.workflow.SignalMethod;
```

Comment:
maybe we can use just the main class here and not include the entire class with includes etc, like we do in other exaples. wdyt?